### DISCUSSION TITLE
How to define the position of the turbine on the Z axis？

### DISCUSSION THREAD

**Discussion by zevenlordsa** [NatLabRockies/floris#720] [Q&A] [answered]
I've checked the Floris.interface.py, and It seems that the turbine's position can only be defined in the x-y axis and not in the Z-axis. Is there any parameter like layout_Z that I haven't seen?
  Thank you very much!

**Comment by Bartdoekemeijer** [comment, marked as answer]
Hi @zevenlordsa. There's no variable like `layout_z` to facilitate this in FLORIS. Instead, you can change each turbine's hub height to mimic varying heights.

**Reply by zevenlordsa** [reply]
oh！That's it. Thank you very much sir!
